bin/sdk-cli.js

```diff
@@ -13,6 +13,9 @@ const path = require( 'path' );
 const yargsModule = require( 'yargs' );
 const webpack = require( 'webpack' );
 
+// This environment variable will be used by the Babel plugin that makes POT files, see babel.config.js.
+process.env.CALYPSO_SDK_OUTPUT_DIR = yargsModule.argv.outputDir;
```

Comment:
To express https://github.com/Automattic/wp-calypso/pull/30303#issuecomment-457191495 in a bit more compact way, how about we move this to `bin/sdk/gutenberg.js` and change it to:
```
const output = outputDir || path.join( inputDir, 'build' );
process.env.CALYPSO_SDK_OUTPUT_DIR = output;
```